relay-general/src/store/normalize/contexts.rs

```diff
@@ -215,6 +225,21 @@ mod tests {
     use super::*;
     use crate::protocol::LenientString;
 
+    #[test]
+    fn test_get_product_name() {
+        let model = "NE2211";
+        let product_name = ANDROID_MAP.get(model).unwrap();
+        assert_eq!(product_name, &"OnePlus 10 Pro 5G");
```

Comment:
nit: I think we can simplify this a bit like you did with the example of `foobar` below:
```suggestion
        assert_eq!(ANDROID_MAP.get("NE2211").unwrap(), &"OnePlus 10 Pro 5G");
```

Reply:
good point! thats definitively better